vuepress/docs/next/tutorials/consume/multitenancy.md

```diff
@@ -52,26 +51,26 @@ sed -i'' 's/\/\/YOUR-APP-NAME\./\/\/YOUR-TENANT-ID\.YOUR-APP-NAME\./g' keycloak-
 
 5. Log in to your [Keycloak admin console](../../docs/consume/identity-management.md#authorization), typically located at `http(s)://YOUR-APPNAME.YOUR-HOST-NAME/auth`.
 
-6. Go to `Select realm` in the top left nav → `Add Realm` → select your `keycloak-realm.json` file. Click `Create`.
+6. Find `Entando` in the left sidebar, below the logo.  Click `Add Realm` and select your `keycloak-realm.json` file. Click `Create`.
 
 7. In the new realm, go to `Clients` → Click the Client ID with `YOUR-APP_NAME`. Under the `Credentials` tab, regenerate the `Secret`. 
 
-> Note: The Secret for this client will be needed in the `entando-tenants-config` Secret below for `YOUR-TENANT-KC-SECRET`.
+> Note: The Secret for this client will be needed in the `entando-tenants-secret.yaml` for `YOUR-TENANT-KC-SECRET` below.
 
-8. Regenerate the Secret for Client ID `YOUR-APP-NAME-de` as well. 
+8. Regenerate the Secret for Client ID `YOUR-APP-NAME-de`. 
 
-9. Go to `Manage` → `Users`. Click `View all users` and choose the `admin` user. [Grant the user Role Mapping](../../docs/consume/identity-management.md#authorization) with these assignments: Client `realm-management` and Role `manage-realm`.
+9. Go to `Manage` → `Users`. Click `View all users` and choose the `admin` user. Go to [Role Mapping](../../docs/consume/identity-management.md#authorization) and make this assignment: Client `realm-management` and Role `manage-realm`.
```

Comment:
Shouldn't we mention that this secret too is required in the entando-tenants-secret.yaml?

Reply:
Yes I was meaning for line 60, sorry. In the entando-tenants-secret.yaml the value generated at line 60 should be put as value of the `deKcClientSecret` property

Then it seems missing to me, but I could have lost it, the newly created realm name should be used as value of the `kcRealm` property

Reply:
@firegloves when I check the entando-tenants-secret.yaml in the samples, there was no deKcClientSecret property. Should this be added in the sample and here? So it looks like this:
"kcClientId": "YOUR-APP-NAME",
"kcClientSecret": "YOUR-TENANT-KC-SECRET",
"KcdeClientSecret": "YOUR-TENANT-KCDE-SECRET"

-will add a note about the kcRealm

Reply:
@jyunmitch which samples are you looking at? yes, they must be fixed so that at least:
- deKcClientId
- deKcClientSecret
are present. while the deKcClientId should be the same across tenants, the deKcClientSecret should be different